src/js/lib/content/src/index.js

```diff
@@ -353,6 +352,10 @@ API.main = async function (inputConfig) {
     let config    = Object.assign({},inputConfig,urlConfig)
     API[globalConfig.windowAppScopeConfigName] = config
 
+    let wasm_url = config.wasm_url ? inputConfig.wasm_url : '/assets/ide.wasm'
+    let wasm_glue_url =
+      config.wasm_glue_url ? inputConfig.wasm_glue_url : '/assets/wasm_imports.js'
+
```

Comment:
inputConfig is already merged with config here! See line `let config    = Object.assign({},inputConfig,urlConfig)`. 
Instead do this:
```js
let defaultConfig = {
    wasm_url = '/assets/ide.wasm',
    wasm_glue_url = '/assets/wasm_imports.js'
}
let config = Object.assign(defaultConfig,inputConfig,urlConfig)
```
And then your config will contain always the rifght thing.

Reply:
fixed